### ISSUE TITLE
[Invalid Response, Waiting for 20s and Retrying...]

### ISSUE THREAD

**Issue by MarilynWuu** [JoshuaChou2018/AutoBA#21]
current task
`Use DESeq2 to perform differential expression analysis to find differentially expressed genes.`

Token indices sequence length is longer than the specified maximum sequence length for this model (4191 > 4096). Running this sequence through the model will result in indexing errors
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:100015 for open-end generation.

[INVALID RESPONSE]
` {'tool': 'DESeq2', 'code': 'Rscript -e "library(DESeq2); counts <- read.delim(\"./examples/case1.1/output/counts.txt\"); dds <- DESeqDataSet(counts, design = ~ condition); dds <- estimateDispersions(dds); res <- lfcShrink(dds); write.csv(res, file = \"./examples/case1.1/output/DESeq2_results.csv\")"'}`
PyDev console: starting.

Python 3.10.0 (default, Mar  3 2022, 09:58:08) [GCC 7.5.0] on linux

[Invalid Response, Waiting for 20s and Retrying...]
invalid response: {'tool': 'DESeq2', 'code': 'Rscript -e "library(DESeq2); counts <- read.delim(\"./examples/case1.1/output/counts.txt\"); dds <- DESeqDataSet(counts, design = ~ condition); dds <- estimateDispersions(dds); res <- lfcShrink(dds); write.csv(res, file = \"./examples/case1.1/output/DESeq2_results.csv\")"'}

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:100015 for open-end generation.

在用本地部署的deepseek-7bi测试case1.1时出现了错误，是因为RAG包安装产生了问题吗？
在之前的问题中我看到大家都有这种问题，这是因为模型本身的问题，是无法解决的吗？
现在项目中推荐的llama-index系列包安装版本会导致运行出现问题，故我安装了0.13版本。如果可以的话可以公开一下安装包的requirements吗
